front/src/components/FilterPanel/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import {NavLink} from 'react-router-dom'
+import '../../CommonStylesheets/formItems.scss'
+import './filterPanel.scss'
+
+const today = new Date();
+var nextDay;
+var dates = [];
+for(var i = 0; i < 9; i++){
```

Comment:
Magic numbers!!!
Why did you decide to use 10 dates? Am I right that it is filter dates? It is worth to use services for such cases. Also you have to be sure that such solution is handy for end user

Reply:
Datepicker